src/splitInteger.test.js

```diff
@@ -1,21 +1,24 @@
-'use strict';
+"use strict";
 
-const splitInteger = require('./splitInteger');
-
-test(`should split a number into equal parts
-  if a value is divisible by a numberOfParts`, () => {
+const splitInteger = require("./splitInteger");
 
+test(`should split a number into equal parts if a value is divisible by numberOfParts`, () => {
+  expect(splitInteger(8, 4)).toEqual([2, 2, 2, 2]);
```

Comment:
There's no check for the function's behavior when the value is not divisible by the number of parts. A test for this scenario is recommended to ensure that the function behaves as expected.